Formula/c/cobo-cli.rb

```diff
@@ -0,0 +1,167 @@
+class CoboCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Cobo Command-line Tool"
+  homepage "https://github.com/CoboGlobal/cobo-cli/blob/master/README.md"
+  url "https://github.com/CoboGlobal/cobo-cli/archive/refs/tags/v0.0.4.tar.gz"
```

Comment:
would be better to use pypi sdist

<img width="1175" alt="image" src="https://github.com/user-attachments/assets/8b1c223b-d5e3-45f2-9455-4058bb7b075f">